### PR TITLE
Updated WebSocket TCK 2.0.1 documentation

### PR DESCRIPTION
**Pull request by jansupol**
Signed-off-by: jansupol <jan.supol@oracle.com>

CC @alwin-joseph @anajosep @arjantijms @cesarhernandezgt @dblevins @m0mus @edbratt @gurunrao @jansupol @jgallimore @kazumura @kwsutter @LanceAndersen @bhatpmk @RohitKumarJain @shighbar @gthoman

### CODE REVIEW

**Comment by jansupol** [comment]
Reflect comments in https://github.com/eclipse-ee4j/jakartaee-tck/issues/701

File: user_guides/websocket/src/main/jbake/content/attributes.conf [hunk 1]
```diff
@@ -20,7 +20,7 @@
 //
 // The environment variable used to specify the home directory
 // for the technology.  Used in config.inc.
-:TechnologyHomeEnv: TS_HOME
+:TechnologyHomeEnv: WEBSOCKET_HOME
```
Comment (gurunrao) [inline user_guides/websocket/src/main/jbake/content/attributes.conf:23]: All references are for TS_HOME, should we change to WEBSOCKET_HOME?

Comment (jansupol) [reply user_guides/websocket/src/main/jbake/content/attributes.conf:23]: `TS_HOME` is named explicitly in the text, TechnologyHomeEnv is the VI home, the home of the vendor implementation. TS_HOME is the TCK Suite home. See other technology TechnologyHomeEnv settings, non seems to be set to TS_HOME.

Comment (jansupol) [reply user_guides/websocket/src/main/jbake/content/attributes.conf:23]: Well, Concurrency, SAAJ, and Servlet have it set to TS_HOME, which is not correct, possibly. The others to VI home, or JAVAEE_HOME.

Comment (jansupol) [reply user_guides/websocket/src/main/jbake/content/attributes.conf:23]: Note that `TechnologyHomeEnv` is used just in `config.inc`.

**Review by gurunrao — APPROVED** [review]

**Pull request merged**